crates/cli-support/src/descriptor.rs

```diff
@@ -67,7 +68,16 @@ pub enum Descriptor {
     String,
     Externref,
     NamedExternref(String),
-    Enum { name: String, hole: u32 },
+    Enum {
+        name: String,
+        hole: u32,
+    },
+    ImportEnum {
+        name: String,
+        invalid: u32,
+        hole: u32,
```

Comment:
Despite my suggestion to remove them from the description, I think it would be nice if we still keep those fields for ease-of-use.